### PR TITLE
Fixed residential external call filters

### PR DESCRIPTION
**Pull request by mmadariaga**
Inject outOfScheduleTargetType = 'number' value (hidden in the form)

#### Type Of Change 
- [X] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx [hunk 1]
```diff
@@ -107,8 +110,8 @@ const properties: ExternalCallFilterProperties = {
   outOfScheduleEnabled: {
     label: _('Out of schedule enabled'),
     enum: {
-      '0': _('No'),
-      '1': _('Yes'),
+      '0': _('Disabled'),
+      '1': _('Enabled'),
```
Comment (Kaian) [inline web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx:114]: If you update this translations, you must also update holidayEnabled field. Those fields are used to toggle the rest of the fields of its section.

File: web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx [hunk 2]
```diff
@@ -224,6 +231,20 @@ const externalCallFilter: EntityInterface = {
...
+    properties: PartialPropertyList,
+    whitelist: string[] = []
+  ): MarshallerValues => {
+    const response = DefaultMarshaller(values, properties, whitelist);
+
+    if (response.outOfScheduleEnabled && !response.outOfScheduleTargetType) {
+      // Force value in residential clients
+      response.outOfScheduleTargetType = 'number';
+    }
+
```
Comment (Kaian) [inline web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx:245]: Right now, outOfScheduleTargetType can be saved without value. Will this affect other clients aside from Residential?

**Review by Kaian — CHANGES_REQUESTED** [review]

File: web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx (hunk 1 above)
Comment (mmadariaga) [reply web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx:114]: I rather revert it. The thing is that outOfScheduleTargetType has different labels in ExternalCallFilters and ExternalCallFiltersResidential

Comment (Kaian) [reply web/portal/client/src/entities/ExternalCallFilter/ExternalCallFilter.tsx]: Ok, revert it and I'll fix that in #2277

**Review by Kaian — APPROVED** [review]

**Pull request merged**